frontend/simple/views/CreateGroup.vue

```diff
@@ -45,6 +35,16 @@
               </span>
             </button>
           </p>
+          <p class="control" v-if="currentStep !== 0">
+            <button
+              class="button is-light is-large"
+              @click="prev"
+              :disabled="!this.currentStep"
```

Comment:
While we're at it... who is fixing this `v-if` / `:disabled` conflict?

Reply:
this is fixed on #390 😄